astra/src/test/java/com/slack/astra/elasticsearchApi/OpenSearchRequestTest.java

```diff
@@ -2,17 +2,27 @@
 
 import static org.assertj.core.api.Assertions.assertThat;
 
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
 import com.google.common.io.Resources;
 import com.slack.astra.logstore.LogMessage;
 import com.slack.astra.proto.service.AstraSearch;
 import java.io.IOException;
 import java.nio.charset.Charset;
 import java.util.List;
 import java.util.Map;
+import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.Test;
 
 public class OpenSearchRequestTest {
 
+  private static ObjectMapper objectMapper;
+
+  @BeforeAll
+  public static void beforeClass() {
+    objectMapper = new ObjectMapper();
```

Comment:
Does this need re-initialized for each test?

Reply:
Ah my bad. I get `@BeforeAll`, `@BeforeEach`, `@Before`, and `@BeforeClass` all confused (the latter two of which are junit 4)